### PR TITLE
-Author Arthur Aghajanyan

### PR DESCRIPTION
**Pull request by ArthurAghajanyan**
-Added React homework 22.05.2019

### CODE REVIEW

File: Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                <TextAndColor name="My" textColor="blue"/>
+                <TextAndColor name="First" textColor="green"/>
+                <TextAndColor name="Project" textColor="orange"/>
+            </div>
+      );
+    }
+}
+class TextAndColor extends ParentComponent {
+    render(){
+        return <h1 class="h1_style" style={{color : this.props.textColor}}>{this.props.name}</h1>;
```
Comment (andreasyanl) [inline Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js]: In jsx we don't have class prop. jsx attribute for class is className .
https://reactjs.org/docs/introducing-jsx.html#specifying-attributes-with-jsx

File: Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                <TextAndColor name="Hello" textColor="red"/>
+                <TextAndColor name="React" textColor="yellow"/>
+                <TextAndColor name="My" textColor="blue"/>
+                <TextAndColor name="First" textColor="green"/>
+                <TextAndColor name="Project" textColor="orange"/>
+            </div>
+      );
+    }
+}
+class TextAndColor extends ParentComponent {
```
[the remark this answers is not in the document]
Comment (ArthurAghajanyan) [reply Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js]: corrected mistakes mentioned comment

File: Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+
+class ParentComponent extends React.Component {
```
[the remark this answers is not in the document]
Comment (ArthurAghajanyan) [reply Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js]: corrected mistakes mentioned comment

File: Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js (hunk 1 above)
Comment (ArthurAghajanyan) [reply Arthur_Aghajanyan/REACT/homework22.05.2019/src/App.js]: corrected mistakes mentioned comment

**Pull request merged**